labs.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: page
+title: "Labs"
+permalink: labs
+---
+
+Solid Labs is a place where institutions that are working on Solid research are listed. If you are interested in being listed on Solid Labs send an email to info@solidproject.org with a description of the research project and website. 
```

Comment:
I think you make Solid Labs too much of a thing.  Prefer "Here are some academic groups specifically doing research around Sold.  If you are interested in being listed on send an email ..."